Resources/Prototypes/Entities/Objects/Specific/chemistry.yml

```diff
@@ -364,6 +364,17 @@
     - Syringe
     - Trash
 
+- type: entity
+  parent: BaseSyringe
+  id: PrefilledSyringe
+  components:
+  - type: Injector
+    toggleState: Inject
+  - type: Tag
```

Comment:
You should remove the `Tag` component entirely so that the tags from the parent prototype get used.

Reply:
I see, didn't really know what the tag component was for. All I know is that the other Syringe entity declared them again so I just followed suit.